libs/form-renderer/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,31 @@
+import { FieldSet, Paragraph, Checkbox as ADSCheckbox, Column } from '@amsterdam/design-system-react'
+
+type CheckboxProps = {
```

Comment:
Just `Props` is fine since it's just used in component's scope. Same for other components.